test/framework/dump.go

```diff
@@ -45,28 +46,37 @@ func (f *CommonFramework) dumpDefaultResourcesInAllNamespaces(ctx context.Contex
 	var result error
 
 	for _, ns := range namespaces.Items {
-		if err := f.dumpEventsInNamespace(ctx, ctxIdentifier, k8sClient, ns.Name); err != nil {
-			result = multierror.Append(result, fmt.Errorf("unable to fetch Events from namespace %s: %s", ns.Name, err.Error()))
-		}
-		if err := f.dumpPodInfoForNamespace(ctx, ctxIdentifier, k8sClient, ns.Name); err != nil {
-			result = multierror.Append(result, fmt.Errorf("unable to fetch information of Pods from namespace %s: %s", ns.Name, err.Error()))
-		}
-		if err := f.dumpDeploymentInfoForNamespace(ctx, ctxIdentifier, k8sClient, ns.Name); err != nil {
-			result = multierror.Append(result, fmt.Errorf("unable to fetch information of Deployments from namespace %s: %s", ns.Name, err.Error()))
-		}
-		if err := f.dumpStatefulSetInfoForNamespace(ctx, ctxIdentifier, k8sClient, ns.Name); err != nil {
-			result = multierror.Append(result, fmt.Errorf("unable to fetch information of StatefulSets from namespace %s: %s", ns.Name, err.Error()))
-		}
-		if err := f.dumpDaemonSetInfoForNamespace(ctx, ctxIdentifier, k8sClient, ns.Name); err != nil {
-			result = multierror.Append(result, fmt.Errorf("unable to fetch information of DaemonSets from namespace %s: %s", ns.Name, err.Error()))
-		}
-		if err := f.dumpServiceInfoForNamespace(ctx, ctxIdentifier, k8sClient, ns.Name); err != nil {
-			result = multierror.Append(result, fmt.Errorf("unable to fetch information of Services from namespace %s: %s", ns.Name, err.Error()))
+		if err := f.DumpDefaultResourcesInNamespace(ctx, ctxIdentifier, k8sClient, ns.Name); err != nil {
+			result = multierror.Append(result, err)
 		}
 	}
 	return result
 }
 
+// DumpDefaultResourcesInNamespace dumps all default K8s resources of a namespace.
+func (f *CommonFramework) DumpDefaultResourcesInNamespace(ctx context.Context, ctxIdentifier string, k8sClient kubernetes.Interface, namespace string) error {
```

Comment:
Is that related to the Testcase introduced in the PR? It seem to me like this is a change to the CommonFramwork instead?

Reply:
you're right. I split the PR and created a new one just for the other changes